app/Controllers/ReservationsAdminController.php

```
@@ -167,10 +167,10 @@ public function update(Request $request, Response $response, mysqli $db, int $id
                 $integrity->recalculateBookAvailability($libroId);
             }
 
-            return $response->withHeader('Location', '/admin/prenotazioni?updated=1')->withStatus(302);
+            return $response->withHeader('Location', '/admin/reservations?updated=1')->withStatus(302);
         } else {
             $stmt->close();
-            return $response->withHeader('Location', '/admin/prenotazioni/modifica/' . $id . '?error=update_failed')->withStatus(302);
+            return $response->withHeader('Location', '/admin/reservations/edit/' . $id . '?error=update_failed')->withStatus(302);
         }
```

PHPMD (2.15.0)

[error] The method update uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)